enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -57,3 +73,34 @@ export const getQuestionParameters = (questionId: CardId) => {
     },
   };
 };
+
+// Define the BackButton property on the InteractiveQuestion component
+const InteractiveQuestion = withPublicComponentWrapper(
+  _InteractiveQuestion,
+) as typeof _InteractiveQuestion & {
+  BackButton: typeof BackButton;
+  FilterBar: typeof FilterBar;
+  Filter: typeof Filter;
+  FilterButton: typeof FilterButton;
+  ResetButton: typeof QuestionResetButton;
+  Title: typeof Title;
+  Summarize: typeof Summarize;
+  SummarizeButton: typeof SummarizeButton;
+  Notebook: typeof Notebook;
+  NotebookButton: typeof NotebookButton;
+  QuestionVisualization: typeof QuestionVisualization;
+};
+
+InteractiveQuestion.BackButton = BackButton;
+InteractiveQuestion.FilterBar = FilterBar;
+InteractiveQuestion.Filter = Filter;
+InteractiveQuestion.FilterButton = FilterButton;
+InteractiveQuestion.ResetButton = QuestionResetButton;
+InteractiveQuestion.Title = Title;
+InteractiveQuestion.Summarize = Summarize;
+InteractiveQuestion.SummarizeButton = SummarizeButton;
+InteractiveQuestion.Notebook = Notebook;
+InteractiveQuestion.NotebookButton = NotebookButton;
+InteractiveQuestion.QuestionVisualization = QuestionVisualization;
```

Comment:
Should we demonstrate how to use these child components in the README, and a brief doc comments on some of the components where it's less obvious what they are? (e.g. I think the SDK user might not know where they should `FilterBar` as opposed to a `Filter`.

Reply:
Yeah I should definitely add a README here, I'll do that in a bit